editors/code/src/inlay_hints.ts

```diff
@@ -39,16 +39,6 @@ export function activateInlayHints(ctx: Ctx) {
     void hintsUpdater.setEnabled(ctx.config.displayInlayHints);
 }
 
-interface InlayHintsParams {
-    textDocument: lc.TextDocumentIdentifier;
-}
-
-interface InlayHint {
-    range: vscode.Range;
```

Comment:
@SomeoneToIgnore what was the reason to declare `vscode.Range` instead of `lc.Range` here? It worked all the time since their shape is structurally the same, but we may not rely on this, so I explicitly added `protocol2CodeConverter.asRange()` call here...

Reply:
No particular reason, pick the one that suits better and works still.